torch/_inductor/compile_fx.py

```diff
@@ -552,7 +566,15 @@ def compile_fx_inner(
     context = torch._guards.TracingContext.try_get()
     if context is not None and context.output_strides is not None:
         assert len(context.output_strides) == 0
-        context.output_strides.extend(compiled_graph.output_strides)
+        shape_env = _shape_env_from_inputs(example_inputs)
+        V.graph.sizevars = SizeVarAllocator(shape_env)
```

Comment:
I don't know if we'd want to assign a SizeVarAllocator to V.graph.sizevars or just use it here? I actually don't know the implications either way... just noting that it was unassigned before.

Reply:
Ah i see, i can do that, but wasnt sure if we needed these sizevars again or not